simulator/base/core/Dockerfile

```diff
@@ -23,7 +23,9 @@ RUN . /etc/os-release \
     && echo "deb http://packages.osrfoundation.org/gazebo/$ID-stable `lsb_release -sc` main" > /etc/apt/sources.list.d/gazebo-latest.list
 
 # install gazebo packages
-RUN apt-get update && apt-get install -y --no-install-recommends \
+RUN apt-get update && apt-get install -y \
+    libgazebo11=11.3.0-1* \
+    gazebo11-common=11.3.0-1*\
     gazebo11=11.3.0-1* \
```

Comment:
The better option here might be to remove the explicit version dependency. This does make the build a little less repeatable but I think it fixes this issue. Seems something odd happens with OSRF's package releases

Reply:
Okay I'll give that a try